server/templates/base.html

```diff
@@ -94,190 +94,31 @@
-                  {# TODO: UNCOMMENT TO RE-ENABLE BIGQUERY #}
-                  {#<a class="dropdown-item" href="https://docs.datacommons.org/bigquery">{% trans %}BigQuery{% endtrans
-                    %}</a>#}
-                  <a class="dropdown-item" href="https://docs.datacommons.org/tutorials">{% trans %}Tutorials{% endtrans
-                    %}</a>
-                  <a class="dropdown-item" href="https://docs.datacommons.org/contributing/">{% trans %}Contribute{%
-                    endtrans %}</a>
-                </div>
-              </li>
-              <li class="nav-item dropdown">
-                <a class="nav-link dropdown-toggle" href="#" id="nav-about-dropdown" role="button"
-                  data-toggle="dropdown" aria-haspopup="true" aria-expanded="false" aria-label="{% trans %}Show about links{% endtrans %}">
-                  {% trans %}About{% endtrans %}
-                </a>
-                <div class="dropdown-menu dropdown-menu-right" aria-labelledby="nav-about-dropdown">
-                  <a class="dropdown-item" href="{{ url_for('static.about') }}">{% trans %}About Data Commons{% endtrans
-                    %}</a>
-                  <a class="dropdown-item" href="https://blog.datacommons.org/">{% trans %}Blog{% endtrans
-                    %}</a>
-                  <a class="dropdown-item" href="https://docs.datacommons.org/datasets/">{% trans %}Data Sources{%
-                    endtrans %}</a>
-                  <a class="dropdown-item" href="{{ url_for('static.faq') }}">{% trans %}FAQ{% endtrans %}</a>
-                  <a class="dropdown-item" href="{{ url_for('static.feedback') }}">{% trans %}Feedback{% endtrans %}</a>
-                </div>
-              </li>
-            </ul>
-          </div>
-        </div>
-      </nav>
-    </header>
+<div id="metadata-base" class="d-none"
+  data-header="{{ HEADER_MENU }}"
+  data-footer="{{ FOOTER_MENU }}"
+  data-logo-path="{{ LOGO_PATH }}"
+  data-name="{{ NAME }}"
+  data-show-disaster="{{ SHOW_DISASTER }}"
+  data-show-sustainability="{{ SHOW_SUSTAINABILITY }}"
+  data-hide-full-footer="{{ is_hide_full_footer }}"
+  data-hide-sub-footer="{{ is_hide_sub_footer }}"
+  data-subfooter-extra="{{ subfooter_extra_content|safe }}"
+  data-brand-logo-light="{{ brand_logo_light }}"
+>
+</div>
+
+{% include 'auxiliary/labels.html' %}
+{% include 'auxiliary/routes.html' %}
 
+<script src={{url_for('static', filename='base.js', t=config['VERSION'])}} async></script>
+  <div id="main">
+    <div id="app-header-container"></div>
     <main id="{{ main_id }}" class="main-content">
       {% block content %}
       {% endblock %}
     </main>
-
-    <footer id="main-footer">
-      {% if not is_hide_full_footer %}
-      <div class="container top-footer">
-        <div class="row">
-          <section class="col-12 col-sm-6 col-md-4">
-            {# TRANSLATORS: The label for a collection of exploration tools. #}
```

Comment:
It's important to keep all of the `{#  TRANSLATORS: ... #}` notes when migrating the `{% trans %}...{% endtrans %}` tags

These tags get extracted by pybabel tool and wind up in the translation files: https://github.com/datacommonsorg/website/blob/f5e8e87c2291d87dfa37a3a887f01d7ff28d6467/server/i18n/all.pot#L77

References:
https://github.com/datacommonsorg/website/blob/master/docs/localization.md
https://github.com/datacommonsorg/website/blob/f5e8e87c2291d87dfa37a3a887f01d7ff28d6467/scripts/extract_messages.sh#L37